server/routers/adRouter.js

```diff
@@ -1,9 +1,30 @@
 const router = require('express').Router();
 const controller = require('../controllers');
 
-router.get('/', controller.admin.get);
-router.post('/', controller.admin.post);
-router.put('/', controller.admin.put);
-router.delete('/', controller.admin.delete);
+router.get('/reservations', controller.reservations.get);
+router.post('/reservations', controller.reservations.post);
+router.put('/reservations', controller.reservations.put);
+router.put('/reservations/:guestId', controller.reservations.put);
```

Comment:
Change PUT route from '/reservations/:guestId' to 'reservations/checkOut/:reservations_id'